specs/discoverable_disk_image.md

```diff
@@ -17,7 +17,18 @@ into one.  They are designed to be composable and stackable, and provide securit
 The images use the GPT partition table verbatim, so it will not be redefined here. Each partition contains
 a standard Linux filesystem (e.g.: `erofs`), so again this will not be redefined here.
 The [DPS](discoverable_partitions_specification.md) defines the GUIDs to use and the format of the
-dm-verity signature partition's JSON content.
+`dm-verity` signature partition's JSON content.
+
+It is recommended to use a sector size of 512 bytes or 4096 for DDIs. Software operating with DDIs should
+automatically derive the sector size used for a DDI by looking for the "EFI PART" magic string at offsets 512
+or 4096, as per GPT specification.
+
+## Naming
+DDIs should use `.raw` as file suffix. A secondary suffix may be used to clarify the specific usage class of
+a DDI. For now the two secondary suffixes `.sysext.raw` and `.confext.raw` are defined (for system extension
```

Comment:
maybe make the text in brackets a link to specs/extension_image.md

Reply:
added a link.